index.js

```diff
@@ -659,18 +659,35 @@ class DHT extends EventEmitter {
   }
 
   async * _resolveBootstrapNodes () {
-    for (const node of this.bootstrapNodes) {
+    for (let { host, port } of this.bootstrapNodes) {
+      const [suggestedIP, fallbackHost] = host.split('@')
+
+      if (fallbackHost) {
+        try {
+          const request = this.io.createRequest({ id: null, host: suggestedIP, port }, null, true, PING)
+          await this._requestToPromise(request)
```

Comment:
pretty sure you can just replace these two lines with a call to this.ping after all, as we jsut use the the default settings